src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -302,7 +306,7 @@ private static double fineTuneTurretRotation(
   private void setTurretPos(double setpoint) {
     turretGoalPositionEntry.log(setpoint);
     turretMotor.set(
-        -turretPIDController.calculate(
+        turretPIDController.calculate(
```

Comment:
Do you need to initialize turretMotor with setInverted()? Or does this work reversed on the real turret vs the proto-turret?

Reply:
addressed in the next commit